notebooks/official/bigquery-ml/bqml-vertexai-model-registry.ipynb

```diff
@@ -0,0 +1,1465 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    PROJECT_ID = "bqml-vertexai-356715"  # @param {type:"string"}
</pre><p>Don't hardcode, see variable substitutions section in go/cloudai-notebook-testing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/842/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Sorry I missed this! Done.</p>